e2e/main.js

```diff
@@ -2,23 +2,51 @@ import '@bpmn-io/form-js/dist/assets/form-js.css';
 import '@bpmn-io/form-js/dist/assets/form-js-editor.css';
 import '@bpmn-io/form-js/dist/assets/form-js-playground.css';
 
-import { FormPlayground } from '@bpmn-io/form-js';
-
-const schema = {
-  schemaVersion: 8,
-  exporter: {
-    name: 'Camunda Web Modeler',
-    version: '0de37f6'
-  },
-  components: [],
-  type: 'default',
-  id: 'form_id',
-  executionPlatform: 'Camunda Cloud',
-  executionPlatformVersion: '8.2.0'
+import './theme.scss';
+
+import { FormPlayground, FormEditor, Form } from '@bpmn-io/form-js';
+import emptySchema from './visual/fixtures/empty.json';
+
+const COMPONENTS = {
+  editor: FormEditor,
+  viewer: Form,
+  playground: FormPlayground,
 };
 
-new FormPlayground({
-  container: document.querySelector('#container'),
-  schema,
-  data: {}
-});
+fetch('/form')
+  .then((response) => {
+    if (response.ok) {
+      return response.json();
+    }
+
+    return Promise.resolve({
+      data: {
+        component: 'playground',
+        schema: emptySchema,
+      },
+    });
+  })
+  .then((response) => {
+    const data = response.data;
+
+    const FormComponent = COMPONENTS[data.component || 'viewer'];
+
+    const form = new FormComponent({
+      container: document.querySelector('#container'),
+      schema: data.schema,
+      data: {},
+    });
+
+    if (data.component !== 'playground') {
+      form.importSchema(data.schema);
+    }
+  })
+  .catch(() => {
+    new FormPlayground({
+      container: document.querySelector('#container'),
+      schema: emptySchema,
+      data: {},
+    });
+  });
```

Comment:
Not sure whether we should swallow all errors. The form component initialization + import could also throw on certain conditions, I think we want to track that.

Reply:
Do you want to have a look or should I?

Reply:
I'll take a look